src/scripting/lua_map.cc

```diff
@@ -8029,10 +8031,10 @@ int LuaField::get_owner(lua_State* L) {
 int LuaField::get_buildable(lua_State* L) {
 	const Widelands::NodeCaps caps = fcoords(L).field->nodecaps();
 	const bool is_buildable =
-	   (caps & Widelands::BUILDCAPS_FLAG) || (caps & Widelands::BUILDCAPS_SMALL) ||
-	   (caps & Widelands::BUILDCAPS_MEDIUM) || (caps & Widelands::BUILDCAPS_BIG) ||
-	   (caps & Widelands::BUILDCAPS_MINE);
-	lua_pushboolean(L, is_buildable);
+	   ((caps & Widelands::BUILDCAPS_FLAG) != 0) || ((caps & Widelands::BUILDCAPS_SMALL) != 0) ||
+	   ((caps & Widelands::BUILDCAPS_MEDIUM) != 0) || ((caps & Widelands::BUILDCAPS_BIG) != 0) ||
+	   ((caps & Widelands::BUILDCAPS_MINE) != 0);
```

Comment:
```suggestion
	   ((caps & (Widelands::BUILDCAPS_FLAG | Widelands::BUILDCAPS_BUILDINGMASK)) != 0);
```
I hope I didn't mess it up. The outermost pair of parentheses are probably not necessary, but they make me more comfortable. ;)